hack/print-workspace-status.sh

```diff
@@ -40,4 +40,9 @@ STABLE_gitTreeState ${KUBE_GIT_TREE_STATE-}
 STABLE_gitVersion ${KUBE_GIT_VERSION-}
 STABLE_gitMajor ${KUBE_GIT_MAJOR-}
 STABLE_gitMinor ${KUBE_GIT_MINOR-}
+gitCommit ${KUBE_GIT_COMMIT-}
```

Comment:
oh. Maybe remove the `STABLE_git` vars then.

do the stamps get re-triggered when the workspace changes? e.g., build `kubectl`, then make your workspace dirty, and build again - is `kubectl` marked dirty? the reason I made the others `STABLE_` was because that was the only way to get updated values linked, at least in the version file.